### ISSUE TITLE
Language selection does not work anymore

### ISSUE THREAD

**Issue by MalteHillmann** [digitalcreations/MaxTo#354]
It is impossible to change to german language. Neither "System default" nor "Deutsch/German" explicit selection works. My OS is running on german language.

**Screenshots**
![maxto-lang](https://user-images.githubusercontent.com/3406318/53558350-266ef200-3b48-11e9-89fb-7333d28600ca.png)

**System information:**
 - Windows version: Windows 10 Pro 18.09 (17763.316)
 - MaxTo version: 2.0.0-rc1

**Additional context**
`%AppData%\MaxTo\maxto.log` contains no data corresponding to this issue.

**Comment by vegardlarsen** [comment]
Does it work if you edit it in `%AppData%\MaxTo\config.json`? The correct ISO code to put for German is `de`.

**Comment by MalteHillmann** [comment]
It is already set to "de":
```
{
  "$schema": "https://maxto.net/config.schema.json",
  "language": "de",
[...]
}
```

**Comment by vegardlarsen** [comment]
Do you have a subfolder in the MaxTo installation directory (`%LocalAppData%\MaxTo\app-2.0.0-rc1`) called `de` with 3 DLLs in it?

**Comment by vegardlarsen** [comment]
This apparently happens in every language. I am investigating this issue.

**Comment by MalteHillmann** [comment]
> Do you have a subfolder in the MaxTo installation directory (`%LocalAppData%\MaxTo\app-2.0.0-rc1`) called `de` with 3 DLLs in it?

Yes:
```
MaxTo.Core.resources.dll
MaxTo.resources.dll
Microsoft.Win32.TaskScheduler.resources.dll
```

**Comment by vegardlarsen** [comment]
I have made headway into figuring out what happened here. It was a issue on the build server, where our tooling did not insert the correct embedded resources (i.e. the translated strings) into the UI project's `.csproj` file. I am rectifying this by adding better tooling.

It worked for Norwegian and English because those languages were not being imported live from POEditor, but were already included in our source repository.

The fix for this will be in 2.0.0-rc.2.

**Comment by vegardlarsen** [comment]
This has now been fixed, and will be in 2.0.0-rc.2.
